Kitodo/src/main/java/org/goobi/production/flow/jobs/HistoryAnalyserJob.java

```diff
@@ -490,16 +480,17 @@ private static Boolean getHistoryEventDuplicated(Process inProcess) {
      * @return Boolean
      */
     public static Boolean updateHistoryForProcess(Process inProc) {
-        Boolean updated = false;
+        Boolean updated;
         try {
+            //TODO: updateHistoryForSteps overwrites result of updateHistory
             updated = updateHistory(inProc);
             updated = updateHistoryForSteps(inProc);
         } catch (Exception ex) {
             logger.warn("Updating history failed.", ex);
+            logger.error(ex.getStackTrace());
```

Comment:
Is this error log call necessary as in the warning call is the stack trace included.

Reply:
It is not needed anymore.